documentation/source/experiment_monitoring.md

```diff
@@ -0,0 +1,103 @@
+# Third-party experiment monitoring
+
+SuperGradients supports out-of-the-box Weights & Biases (wandb) and ClearML. 
+
+
+Simply update your train parameters to specify your preferred third-party logging tool, and the Trainer will 
+take care of the rest, handling all the necessary logic and integration.
+
+### Tensorboard
+**requirements**: None
+
+Tensorboard is natively integrated into the training and validation steps. You can find how to use it in [this section](TODO:add_link_to_our_tn_description).
+
+
+### Weights & Biases
+**requirements**:
+- Install `wandb`
+- Set up wandb according to the [official documentation](https://docs.wandb.ai/quickstart#1.-set-up-wandb)
+- Adapt your code like in the following example
+
+```python
+from super_gradients import Trainer
+
+trainer = Trainer("experiment_name")
+model = ...
+
+training_params = {
+    ...                             # Your training params
+    "sg_logger": "wandb_sg_logger", # Weights&Biases Logger, see class super_gradients.common.sg_loggers.wandb_sg_logger.WandBSGLogger for details
+    "sg_logger_params":             # Params that will be passes to __init__ of the logger super_gradients.common.sg_loggers.wandb_sg_logger.WandBSGLogger
+      {
+        "project_name": "project_name", # W&B project name
```

Comment:
```
entity:
api_server:   # in case your experiment tracking is not hosted at wandb servers
```

Reply:
Added in 5168bb65f56074a9aa1cac07d0a28b19a55c7107